dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -18,11 +18,17 @@ import '../framework/host_agent.dart';
 import '../framework/task_result.dart';
 import '../framework/utils.dart';
 
-/// Must match flutter_driver/lib/src/common.dart.
-///
-/// Redefined here to avoid taking a dependency on flutter_driver.
-String _testOutputDirectory(String testDirectory) {
-  return Platform.environment['FLUTTER_TEST_OUTPUTS_DIR'] ?? '$testDirectory/build';
+
+void copyToOutputLogs(String outputLog)
+{
+  final String? outputDirectory = Platform.environment['FLUTTER_TEST_OUTPUTS_DIR'];
+  /// Noop if `FLUTTER_TEST_OUTPUTS_DIR` is not set.
```

Comment:
nit: this is just a regular code comment, so it should be

```suggestion
  // Noop if `FLUTTER_TEST_OUTPUTS_DIR` is not set.
```